cognite/neat/rules/models/_rules/_types/_base.py

```diff
@@ -411,3 +357,60 @@ def from_raw(cls, value: Any) -> Self:
 
     def as_class_entity(self) -> ClassEntity:
         return ClassEntity(prefix=self.prefix, suffix=self.suffix, version=self.version)
+
+
+class ReferenceEntity(ViewPropEntity):
```

Comment:
This is a simple move?

Reply:
Yes simple move, to avoid linter to complain.